SimulationAgent/Agent.cs

```diff
@@ -192,6 +196,44 @@ private async Task RunAsync()
             }
         }
 
+        // This creates sample simulations that will be shown on simulation dashboard by default
+        public async Task SeedAsync(string templateName)
+        {
+            string content;
+            var fileName = templateName + ".json";
+            var root = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
+            var file = Path.Combine(root, "Data", fileName);
```

Comment:
note that `Data` and `data` differs depending on the OS. By default we use lowercase paths

Reply:
changed to data